### ISSUE TITLE
add chronophotography effect

### ISSUE THREAD

**Issue by loopier** [loopier/animatron#60]
On every frame, add lighter pixels

After Roland's chronomocrogeaphy

**Comment by totalgee** [comment]
What is "chronomocrogeaphy"? Do you have a link? I do know what [chronophotography](https://en.wikipedia.org/wiki/Chronophotography) is, though, so I think maybe I know what you mean.

You want a shader effect that would do this? (it would require feedback, since you're using the previous frame(s))

**Comment by loopier** [comment]
Yes, it's the same, just applied to microscope photography. But is the effect that's cool. It's applicable to both and more.

Haven't thought how. The easiest or coolest. If you want to go for it, please do!
